orm/caching.md

```diff
@@ -1 +1,19 @@
 # Caching
+
+This package supports these caching systems out of the box:
+
+* `redis`
+* `memcached`
+* `file`
+* `acp`
+* `array`
+
+## Extending or Adding Cache Drivers
+
+Drivers can be replaced or added using `CacheManager`
+
+```php
+CacheManager::extend('customCache' function() {
+    return 'customCacheProvider'
```

Comment:
@patrickbrouwers Can you provide a more complete example of creating a new cache provider?